docs/wizard.md

```diff
@@ -0,0 +1,102 @@
+### The wizard, or start command
+
+Once running the command you should first see one of the prompts below:
+
+- if you don't have an active `project`.
+
+```shell
+Welcome to the Affinidi Wizard
+You are authenticated as: <your-email@address.com>
+Active project: no active projects
+Please enter a project name:
+```
+
+- if you already have an active `project`.
+
+```shell
+> Welcome to the Affinidi Wizard
+You are authenticated as: <your-email@address.com>
+Active project: <project-id> <project-name>
+
+? select your next step (Use arrow keys)
+❯ manage projects
+  manage schemas
+  generate an application
+  issue a vc
+  verify a vc
+  logout
+  exit
+```
+
+Where:
+
+- `manage projects` brings you to use any command related to `projects`:
+
+  - creating a new one
+  - seeing the active one
+  - activating another one
+  - listing them all
+  - seeing the details of one in particular
+
+  Output when selecting this option:
+
+  ```shell
+  ? select your next step (Use arrow keys)
+  ❯ change active project
+    create another project
+    show active project
+    show project's details
```

Comment:
here the the 's in projects needs an escape character

Reply:
not sure, because it renders the backslash otherwise
<img width="600" alt="image" src="https://user-images.githubusercontent.com/86773100/216247697-f5886839-8d69-4c83-a4a7-98a0bdbeb6ed.png">